Test.C

```diff
@@ -235,6 +235,57 @@ int main(){
         // Selling equipment
         // decrease amount in index amount if things are sold
         // increase credit if things are sold
+        // So this is a lot of pseudo code or not finished but idk how to do it fully but it a good framework 
+        // assumes that you can sell the equipment for the same price that is bought 
+        int equipSellSelect;
+        int equipSellAmount;
+        int equipSellTransaction;
+        int equipSellAnswer;
+        
+        printf("|----------------------------- Equipment Selling ------------------------------|\n");
+        printf("Equipment that you own:\n");
+        for(int i = 0; i < NUM_EQUIPMENT; i++){
+            printf("%s %s %d %d\n", equipmentStock[i].name, equipmentStock[i].description, equipmentStock[i].cost, equipmentStock[i].messageAmount);
+            printf("Amount of %s that is in your inventory: %d", equipmentStock[i].name, player.equipmentInventory[i]);
```

Comment:
Don't forget the newline character.
```suggestion
            printf("Amount of %s that is in your inventory: %d\n", equipmentStock[i].name, player.equipmentInventory[i]);
```